README.md

```diff
@@ -4,6 +4,52 @@ status: just brainstorming
 
 [![Build Status](https://travis-ci.org/liqd/aula.svg?branch=master)](https://travis-ci.org/liqd/aula)
 
+
+## Getting started (with docker)
+
+- install docker
+- docker pull fisx/aula
+- git checkout aula
+- cd aula
+- docker run --rm -it -p 8080:8080 -v `pwd`:/root/aula fisx/aula /bin/bash
+- inside container:
+    - cd /root/aula
+    - cabal init --sandbox=/liqd/thentos/.cabal-sandbox
+    - perl -i -pe s/"127.0.0.1"/"0.0.0.0"/ src/Config.hs
+    - cabal run aula-server
+- then point your browser to localhost:8080
+
+if you want to hack both thentos and aula, `/root/thentos` is exposed
+as a docker volume similarly to `/root/aula`.  Just duplicate the `-v`
+option to `docker run`.
+
+
+## Getting started (with sensei)
+
+https://github.com/hspec/sensei is a tool for rapid re-compilation and
+testing.  It works with .  If you want to use it, follow these steps:
```

Comment:
@fisx: did you fix it? you said "11:55 < fisx_> npouillard: not sure what that sentence was for.  i removed it.  and added one more paragraph." but perhaps you didn't push?